README.md

```diff
@@ -157,9 +151,13 @@ You are welcome contribute to App Manager! This doesn't mean that you need codin
 38bzvWDD99dJhXg9tC4yQEnGdnAKPtwSXG
 3FHTxPoYa92dNJK6pkhwyVkMG8Vv3VpGpg
 ```
-By sending me BTC, you agree that you will not share the transaction information in public i.e. the transaction will remain anonymous nor will you use it as a leverage to prioritise your requested features. I accept feature requests without any donations, and they are prioritised according to my preferences.
+By sending me BTC, you agree that you will not share the transaction info in public i.e. the transaction will remain anonymous, nor will you use it as a leverage to prioritise your requested features. I accept feature requests without any donations, and they are prioritised according to my preferences.
 
-**App Manager is open for a fund/grant.** If you are an organization interested in funding App Manager, you can contact me directly at muntashirakon [at] riseup [dot] net (FINGERPRINT: `7bad37c2981e41f8f6abea7f58f0b4f26c346fce`).
+**App Manager is open for funding/grants.** If you are an organisation interested in funding it you can contact me directly at muntashirakon [at] riseup [dot] net (FINGERPRINT: `7bad37c2981e41f8f6abea7f58f0b4f26c346fce`).
 
 ### Credits and Libraries
 A list of credits and libraries are available in the **About** section of the app.
+
+### Community
```

Comment:
I think it's better to put it near translations or contributing since they're related.